Resources/Prototypes/_StarLight/Catalog/uplink_catalog.yml

```
@@ -156,6 +156,15 @@
      state: mag-1 
     }
   productEntity: MagazineMagnumRifleSP
+  cost:
+    Telecrystal: 2
+  categories:
+    - UplinkAmmo
+  conditions:
+    - !type:StoreWhitelistCondition
+    whitelist:
+      tags:
+        - NukeOpsUplink
```

```suggestion
    - !type:StoreWhitelistCondition
      whitelist:
        tags:
          - NukeOpsUplink
```

I made a mistake, sorry